suite-native/module-device-settings/src/constants.tsx

```diff
@@ -0,0 +1,23 @@
+import { CardStepperMap } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+
+export const cardStepperContentMap = {
```

Comment:
nitpick: I would call it `wipeDeviceCardStepperContentMap` or `wipeDeviceStepToContentMap` and I would name the file the same instead of general constants.